openmdao/error_checking/tests/test_check_config.py

```diff
@@ -81,13 +77,10 @@ def test_dataflow_1_level(self):
         # Conclude setup but don't run model.
         p.final_setup()
 
-        warnings = testlogger.get('warning')
-        info = testlogger.get('info')
-        self.assertEqual(len(warnings), 1)
-        self.assertEqual(len(info), 1)
-
-        self.assertEqual(info[0], "The following groups contain cycles:\n   Group '' has the following cycles: [['C1', 'C2', 'C4']]\n")
-        self.assertEqual(warnings[0], "The following systems are executed out-of-order:\n   System 'C3' executes out-of-order with respect to its source systems ['C4']\n")
+        self.assertTrue(testlogger.contains_regex('info',"The following groups contain cycles.*"))
```

Comment:
checking for "The following groups contain cycles"  or "The following systems are executed out-of-order" is fine, but we also need to check for the specific systems that are in the cycle or that are out of order.  Otherwise the checks could be working incorrectly and we wouldn't know it.

Reply:
good point. Will fix.